.github/workflows/ci.yml

```diff
@@ -140,6 +140,23 @@ jobs:
           cd gopath/src/github.com/google/syzkaller
           .github/workflows/run.sh make presubmit_race
 
+  race_dashboard:
+    runs-on: arc-runner-set
+    container: gcr.io/syzkaller/env:latest
+    env:
+      GOPATH: /__w/syzkaller/syzkaller/gopath
+      CI: true
+      TERM: dumb
+    steps:
+      - name: checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          path: gopath/src/github.com/google/syzkaller
+      - name: run
+        run: |
+          cd gopath/src/github.com/google/syzkaller
+          .github/workflows/run.sh make presubmit_race_dashboard
```

Comment:
There's also `SYZ_SKIP_DASHBOARD=1` (grep `Makefile`). I think we don't need both approaches.

Reply:
Nice catch.
Existing approach is to compile files, run tests and skip test if it requires dev_appserver.py.

Proposed approach require every ./dashboard/app/*_test.go file to have an opt-out header and excludes files from the compilation.
@dvyukov do you mind to remove SYZ_SKIP_DASHBOARD env var and use go:build directive instead?

Reply:
What are pros and cons?
SYZ_SKIP_DASHBOARD already exists and don't pollute every test file.
What are advantages of the build directive?

Reply:
SYZ_SKIP_DASHBOARD wins because it already works.
Its usage in race_dashboard improves both, build time and logs readability.

It was quite hard to understand the tests=>ci_jobs mapping because of some duplication.
I expected to see the path based of feature based separation.
What we do is an opt-out of dev_appengine.py tests from build job and run all dashboard tests (including dev_appserver.py) on dashboard job.
One more commit here renames SYZ_SKIP_DASHBOARD to SYZ_SKIP_DEV_APPSERVER_TESTS.

WDYT?

Reply:
SYZ_SKIP_DEV_APPSERVER_TESTS looks reasonable to me